Uchu.World/Scripting/Native/ObjectScript.cs

```diff
@@ -0,0 +1,485 @@
+                    this.SendEffectsToPlayer(player);
+                });
+            }
+            Listen(this.Zone.OnPlayerLoad, (player) =>
+            {
+                Listen(player.OnReadyForUpdatesEvent, (message) =>
+                {
+                    if (message.GameObject != this.GameObject) return;
+                    this.SendEffectsToPlayer(player);
+                });
+            });
+        }
+        
+        /// <summary>
+        /// Callback that is run once with the first GameObject created.
+        /// </summary>
+        public virtual void CompleteOnce()
+        {
+            
+        }
+
+        #region Object
+        /// <summary>
+        /// Destroys the object.
+        /// </summary>
+        /// <param name="killerId">Killer of the object.</param>
+        public void Die(Player killerId)
+        {
+            Task.Run(async () =>
+            {
+                await this.GameObject.GetComponent<DestructibleComponent>().SmashAsync(killerId);
+            });
+        }
+        #endregion
+
+        #region Variable Storage
+        /// <summary>
+        /// Sets a stored variable of the script.
+        /// </summary>
+        /// <param name="name">Name of the variable to store.</param>
+        /// <param name="value">Value of the variable to store.</param>
+        public void SetVar(string name, object value)
+        {
+            this._variables[name] = value;
+        }
+        
+        /// <summary>
+        /// Sets a networked variable of the script.
+        /// </summary>
+        /// <param name="name">Name of the variable to store.</param>
+        /// <param name="value">Value of the variable to store.</param>
+        public void SetNetworkVar(string name, object value)
+        {
+            // Set the variable.
+            this.SetVar(name, value);
+            
+            // Broadcast the variable change.
+            Task.Run(() =>
+            {
+                this.Zone.BroadcastMessage(new ScriptNetworkVarUpdateMessage()
+                {
+                    Associate = this.GameObject,
+                    LDFInText = new LegoDataDictionary()
+                    {
+                        {name, value}
+                    }.ToString(),
+                });
+            });
+        }
+        
+        /// <summary>
+        /// Sets a networked variable of the script.
+        /// </summary>
+        /// <param name="name">Name of the variable to store.</param>
+        /// <param name="value">Value of the variable to store.</param>
+        /// <param name="type">Type of the LDF entry to send.</param>
+        public void SetNetworkVar(string name, object value, byte type)
+        {
+            // Set the variable.
+            this.SetVar(name, value);
+            
+            // Broadcast the variable change.
+            Task.Run(() =>
+            {
+                Console.WriteLine("SET: " + new LegoDataDictionary()
```

Comment:
This `Console.WriteLine` could be changed to `Logger.Debug` for consistency, or removed if it isn't necessary/useful anymore to keep it in

Reply:
This line was accidentally committed when I was testing. In general, if there is a `Console.WriteLine`, it is something I accidentally committed instead of me not using `Logger.Debug`.